src/index.html

```diff
@@ -7,6 +7,295 @@
     <link rel="stylesheet" href="./styles/main.css">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="" class="header__logo">
+        <img src="./images/logo.png" alt="">
+      </a>
+      <nav class="navigation">
+        <ul class="list navigation__list">
+          <li class="list__item">
+            <a class="list__item-link is-active" href="">Apple</a>
+          </li>
+          <li class="list__item">
+            <a class="list__item-link" href="">Samsung</a>
+          </li>
+          <li class="list__item">
+            <a class="list__item-link" href="">Смартфоны и телефоны</a>
+          </li>
+          <li class="list__item">
+            <a
+              class="list__item-link"
+              href=""
+              data-qa="nav-hover"
+              >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="list__item">
+            <a class="list__item-link" href="">Гаджеты</a>
+          </li>
+          <li class="list__item">
+            <a class="list__item-link" href="">Планшеты</a>
+          </li>
+          <li class="list__item">
+            <a class="list__item-link" href="">Фото</a>
+          </li>
+          <li class="list__item">
+            <a class="list__item-link" href="">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="card main__card" data-qa="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="iMac"
+          class="card__photo"
+        >
+        <h3 class="card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h3>
+        <p class="card__small-text card__code">Код товара: 195434</p>
+        <div class="card__rating rating__stars--4">
```

Comment:
- DOM element cannot have 2 BEM elements class names
- `rating__stars--4` should be inside `rating` BEM block
```suggestion
        <div class="card__rating rating rating--4">
```